Assets/Mirror/Runtime/Transport/Websocket/Server.cs

```diff
@@ -76,6 +76,10 @@ public WebSocket GetClient(int connectionId)
 
         public SslConfiguration _sslConfig;
 
+        Queue<ArraySegment<byte>> messagesToSend = new Queue<ArraySegment<byte>>();
```

Comment:
Can you add the messagesToSend queue per client instead?

If we have 10 messages to 10 different clients,  they should all overlap.  The problem is having 10 messages to the the same client